src/containers/BooksList.js

```diff
@@ -2,37 +2,50 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
-import { removeBook } from '../actions/index';
+import { removeBook, changeFilter } from '../actions/index';
+import CategoryFilter from '../components/CategoryFilter';
 
 class BooksList extends Component {
   constructor(props) {
     super(props);
     this.handleRemoveBook = this.handleRemoveBook.bind(this);
   }
 
+  handleFilterChange = event => {
```

Comment:
Guess we need to bind `handleFilterChange` right?

Or is there a new update in React I am not aware of?